packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx

```diff
@@ -15,10 +14,10 @@ const WardPatientCard: React.FC<WardPatientCardProps> = (props) => {
         <WardPatientCardRow key={i} {...props} />
       ))}
       <button
-        className={styles.wardPatientCardButton}
-        onClick={() =>
-          launchWorkspace<WardPatientWorkspaceProps>('ward-patient-workspace', { patientUuid: props.patient.uuid })
-        }>
+        className={classNames(styles.wardPatientCardButton, {})}
```

Comment:
If we don't have any conditions to take care in the classes, we can ignore using classnames.